packages/docusaurus-theme-classic/src/theme/BlogListPage/index.tsx

```diff
@@ -54,6 +55,7 @@ export default function BlogListPage(props: Props): JSX.Element {
         ThemeClassNames.page.blogListPage,
       )}>
       <BlogListPageMetadata {...props} />
+      <BlogListPageStructuredData {...props} />
```

Comment:
Is there an officially recommended place to render that script tag or can it be rendered anywhere?

Wouldn't it be better to add it to the `<head>`?

Reply:
I believe the advice is to colocate the tag and not to put it in the head - I'll see if I can find the link.

Reply:
Can't find the link that I remembered advising colocation. Google doesn't express an opinion that I can see:

> A JavaScript notation embedded in a `<script>` tag in the `<head>` and `<body>` elements of an HTML page. 

https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data#supported-formats